ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -263,12 +265,15 @@ Query Model With GRPCURL
         END
         ${cmd}=    Catenate    ${cmd}    -cacert ${cert}
     END
+    IF    "${protobuf_file}" != "${NONE}"
+        ${cmd}=    Catenate    ${cmd}    -proto ${protobuf_file}
+    END
     IF    ${plaintext}
         ${cmd}=    Catenate    ${cmd}    -plaintext
     END
-    # IF    "${proto_filepath}" != "${NONE}"
-    #     ${cmd}=    Catenate    ${cmd}    -proto ${proto_filepath}
-    # END
+    IF    "${input_filepath}" != "${NONE}"
+         ${cmd}=    Catenate    ${cmd}    < ${input_filepath}
+     END
```

Comment:
cannot you use "@<filepath>" like done for REST? https://github.com/red-hat-data-services/ods-ci/pull/1790/files#diff-f00e22a70bddf287c7c82941822df62d8057a8112d20702ff1bebcdf08398a26R22

Reply:
I tried it, but it doesn't work that way, so I need to use this method instead